test/compression.js

```diff
@@ -9,6 +9,13 @@ var zlib = require('zlib')
 
 var compression = require('..')
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = 'createBrotliCompress' in zlib
+var brotlit = hasBrotliSupport ? it : it.skip
```

Comment:
A test for non-brotli support that ensures it still prefers gzip?

Reply:
done in [af99def](https://github.com/expressjs/compression/pull/194/commits/af99def627a41c21a60b392d66f2dbaf61ec34fe)